src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/FromKeyedServicesAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public class FromKeyedServicesAttribute : Attribute
```

Comment:
sealed?

Reply:
Is that a good idea though? Could that interfere with future source generators? We could always shoot of a quick email to get API approval for sealing it. It seems safer since we could always unseal later.